MDE-EdFiClientSDK/EdFi/generate-sdk.ps1

```diff
@@ -22,9 +22,8 @@ $params = @(
 )
 
 & $javaExecutable $params -i "$metaDataUrl/data/v3/descriptors/swagger.json"                                               --api-package Apis.Descriptors                                                  --model-package Models.Descriptors
-& $javaExecutable $params -i "$metaDataUrl/identity/v2/swagger.json"                                                       --api-package Apis.Identity                                                     --model-package Models.Identity
 & $javaExecutable $params -i "$metaDataUrl/data/v3/profiles/minnesota-preview-sisvendorprofile/swagger.json"               --api-package Apis.Profiles.Minnesota_Preview_SISVendor_Profile                 --model-package Models.Profiles.Minnesota_Preview_SISVendor_Profile
-& $javaExecutable $params -i "$metaDataUrl/data/v3/profiles/minnesota-twentyone-twentytwo-sisvendorprofile/swagger.json"   --api-package Apis.Profiles.Minnesota_Twenty_One_Twenty_Two_SISVendor_Profile   --model-package Models.Profiles.Minnesota_Twenty_One_Twenty_Two_SISVendor_Profile
+& $javaExecutable $params -i "$metaDataUrl/data/v3/profiles/minnesota-twenty-two-twenty-three-baseline-sis-vendor-profile/swagger.json"   --api-package Apis.Profiles.Minnesota_Twenty_One_Twenty_Two_SISVendor_Profile   --model-package Models.Profiles.Minnesota_Twenty_One_Twenty_Two_SISVendor_Profile
```

Comment:
@jolivas-dl , I would expect the api and model namespaces on the profile to match the profile name.  If there is a reason to keep these as is, I am open to discussion.

Reply:
These names have been adjusted to suggested standard.